pkg/deploy/dev-workspace-config/dev_workspace_config_test.go

```diff
@@ -85,7 +87,10 @@ func TestReconcileDevWorkspaceConfigPerUserStorage(t *testing.T) {
 					DevEnvironments: chev2.CheClusterDevEnvironments{},
 				},
 			},
-			expectedOperatorConfig: &controllerv1alpha1.OperatorConfiguration{},
+			// The workspace config is now created by default since we need to enable or disable pod/container security contexts depending on Che Cluster CR
```

Comment:
This comment should be removed, I just added it to explain why an existing test needed to be modified to pass.